### PR TITLE
Replaced interpreter.rs with interpreter_new.rs

### PR DESCRIPTION
**Pull request by erkska**
Issue #37 
Removed `interpreter.rs` and renamed `interpreter_new.rs` to that.

### CODE REVIEW

**Comment by gussmith23** [comment]
Please take a look at the build errors -- there are some more code changes necessary!

**Comment by erkska** [comment]
OK?

**Comment by gussmith23** [comment]
Looks like there are some conflicts! The interpreter probably changed since you started working on this. Can you rebase your changes?

File: src/language/mod.rs [hunk 1]
```diff
@@ -3,8 +3,8 @@ mod language;
 pub use language::*;
 
 //pub mod interpreter;
-pub mod interpreter_new;
-pub use interpreter_new as interpreter;
+pub mod interpreter;
+pub use interpreter;
```
Comment (gussmith23) [inline src/language/mod.rs:7]: We can just delete this line; it doesn't do anything anymore. `pub use .. as ..` allowed us to rename the module, but we don't need to do that anymore!

**Comment by erkska** [comment]
huh, I hope I made it correctly

**Comment by gussmith23** [comment]
Thanks @erkska!

**Pull request merged**